### DISCUSSION TITLE
why

### DISCUSSION THREAD

**Discussion by maska228** [community/community#20148] [Other Feature Feedback, Questions, & Ideas] [answered]
# why
just a genuine "why"
(read tldr unless you want to lose a braincell)
why would i want people to get recommended to me? this isn't youtube.
why did you even implement this? who wants to know what other devs code in lua, and are interested in minecraft clients? like, just why

im here to ~publicly show how bad of a dev i am~ look at my shitty code, and to look at code of other people who develop using programming languages that i atleast vaguely know or can read and understand. im not here to get recommendations for repositories i don't give a flying donald d. duck about.

i already am in a bad mood from youtubes(and googles by extent) shity handling of the bot situation, but here is github, publishing an __ALL NEW FEATURE__ that __EVERYONE CARES ABOUT__

this is kinda just baseless screaming at this point so tldr below summarizes what i wanted to say
# tldr
the feed sucks
doesnt show ppl that i actually would care about+shows alot of people whose repos ive visited a long time ago
idgaf about the feed and its not related at all to people i follow, so i wont get a use out of it anyways.

#

does this need a "/imnotjokingyoutwitteruser" tone indicator, or is it obvious enough?
# :wq

**Comment by maska228** [comment, marked as answer]
🤡

**Reply by maska228** [reply]
yes

**Comment by cardison** [comment]
=)
